test/python/test_qcvv_fitters.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+# pylint: disable=missing-docstring
```

Comment:
Can you remove this disable and add human-friendly docstrings, following the ones in the other test files as a guideline? Some of them might end up being a bit redundant, but the idea is to improve maintainability and give context to the future readers of the file . For example:

```
def test_exp_fit_fun(self):
    """Test exponential decay fitter."""
```

Even with that little change (and no need to go overboard with the docstrings!), readability is increased, specially for  non-experts in the domain (me included :grimacing: ).

Actually, it might be even better to go a bit further and rename the test function for increasing clarity (ie. `def test_fitter_exponential_decay()` or similar) - this way we also decouple "what we want to test" (a particular fitter as a whole) from the details of how that is implemented (in this case, via a `exp_fit_fun()` function, but leaves room for changing the implementation in the future).